docs/application/web/api/6.5/device_api/mobile/tizen/ml_single.html

```diff
@@ -0,0 +1,763 @@
+</pre>
+</div>
+</dd>
+<dt class="method" id="SingleShot::setTimeout">
+<a class="backward-compatibility-anchor" name="::MLSingleShot::SingleShot::setTimeout"></a><code><b><span class="methodName">setTimeout</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Sets the maximum amount of time to wait for an output from <a href="#SingleShot::invoke">SingleShot::invoke()</a> method, in milliseconds.
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void setTimeout(unsigned long timeout);</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">timeout</span>:
+ The time to wait for an output.
+                </li>
+        </ul>
+</div>
+<div class="exceptionlist">
+<p><span class="except">Exceptions:</span></p>
+          <ul class="exception"><li>WebAPIException<ul>
+<li class="list"><p>
+ with error type NotSupportedError, if this feature is not supported, or given property is not available.
+                </p></li>
+<li class="list"><p>
+ with error type AbortError, if any other error occurs.
+                </p></li>
+</ul>
+</li></ul>
+        </div>
+<div class="example">
+<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">var model = tizen.ml.single.openModel(
+    "/opt/usr/home/owner/media/Documents/mobilenet_v1_1.0_224_quant.tflite");
+
+/* 1 ms is used only for testing, to show TimeoutError */
+/* Use more reasonable value in real life applications */
+model.setTimeout(1);
+
+var tensorsInfo = new tizen.ml.TensorsInfo();
+tensorsInfo.addTensorInfo("tensor", "UINT8", [3, 224, 224]);
+var tensorsData = tensorsInfo.getTensorsData();
+try
+{
+  var tensorsDataOut = model.invoke(tensorsData);
+
+  /* Always call dispose() on no longer needed objects */
+  tensorsDataOut.dispose();
+}
+catch (e)
+{
+  console.log("Error: " + e.message);
+}
+
+/* Always call dispose() on no longer needed objects */
+tensorsData.dispose();
+tensorsInfo.dispose();
+model.close();
+</pre>
+</div>
+<div class="output">
+<span class="title"><p>Output example:</p></span><pre>Error: Failed to invoke: timeout
+</pre>
+</div>
+</dd>
+<dt class="method" id="SingleShot::close">
+<a class="backward-compatibility-anchor" name="::MLSingleShot::SingleShot::close"></a><code><b><span class="methodName">close</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Closes the model and releases memory
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void close();</pre></div>
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="exceptionlist">
+<p><span class="except">Exceptions:</span></p>
+          <ul class="exception"><li>WebAPIException<ul><li class="list"><p>
+ with error type AbortError, if any error occurs. Calling methods on closed model will trigger <var>AbortError</var>.
```

Comment:
Not clear 

with error type AbortError, if any error occurs, calling methods on closed model will trigger <var>AbortError</var>.

Reply:
I moved this to remark.